Extensions/xkit_preferences.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE XKit Preferences **//
-//* VERSION 5.0.0 **//
+//* VERSION 5.0.1 **//
```

Comment:
This is adding functionality, so [per semver](http://semver.org/) should be a minor patch bump.